src/DssSpell.sol

```diff
@@ -16,135 +16,461 @@
+        DssExecLib.deauthorize(addr, OLD_MCD_ESM);
+        DssExecLib.authorize(addr, NEW_MCD_ESM);
+
+        // OPTIMISM_ESCROW
+        addr = DssExecLib.getChangelogAddress("OPTIMISM_ESCROW");
+        DssExecLib.deauthorize(addr, OLD_MCD_ESM);
+        DssExecLib.authorize(addr, NEW_MCD_ESM);
+
+        // OPTIMISM_GOV_RELAY
+        addr = DssExecLib.getChangelogAddress("OPTIMISM_GOV_RELAY");
+        DssExecLib.deauthorize(addr, OLD_MCD_ESM);
+        DssExecLib.authorize(addr, NEW_MCD_ESM);
+
+        // ARBITRUM_DAI_BRIDGE
+        addr = DssExecLib.getChangelogAddress("ARBITRUM_DAI_BRIDGE");
+        DssExecLib.deauthorize(addr, OLD_MCD_ESM);
+        DssExecLib.authorize(addr, NEW_MCD_ESM);
+
+        // ARBITRUM_ESCROW
+        addr = DssExecLib.getChangelogAddress("ARBITRUM_ESCROW");
+        DssExecLib.deauthorize(addr, OLD_MCD_ESM);
+        DssExecLib.authorize(addr, NEW_MCD_ESM);
+
+        // ARBITRUM_GOV_RELAY
+        addr = DssExecLib.getChangelogAddress("ARBITRUM_GOV_RELAY");
+        DssExecLib.deauthorize(addr, OLD_MCD_ESM);
+        DssExecLib.authorize(addr, NEW_MCD_ESM);
+
+        // MCD_JOIN_DIRECT_AAVEV2_DAI
+        addr = DssExecLib.getChangelogAddress("MCD_JOIN_DIRECT_AAVEV2_DAI");
+        DssExecLib.deauthorize(addr, OLD_MCD_ESM);
+        DssExecLib.authorize(addr, NEW_MCD_ESM);
+
+        DssExecLib.setChangelogAddress(MCD_ESM_NAME, NEW_MCD_ESM);
+        DssExecLib.setChangelogVersion("1.10.0");
+
+        //////////////////////////////////////////////////////////
+        // Delegate Compensation January Distribution
+        // https://forum.makerdao.com/t/recognized-delegate-compensation-breakdown-january-2022/13001
+
+        DssExecLib.sendPaymentFromSurplusBuffer(FLIP_FLOP_FLAP_WALLET,  12_000);
+        DssExecLib.sendPaymentFromSurplusBuffer(FEEDBLACK_LOOPS_WALLET, 12_000);
+        DssExecLib.sendPaymentFromSurplusBuffer(SCHUPPI_WALLET,         12_000);
+        DssExecLib.sendPaymentFromSurplusBuffer(MAKERMAN_WALLET,         8_620);
+        DssExecLib.sendPaymentFromSurplusBuffer(MONETSUPPLY_WALLET,      4_807);
+        DssExecLib.sendPaymentFromSurplusBuffer(ACRE_INVEST_WALLET,      3_795);
+        DssExecLib.sendPaymentFromSurplusBuffer(JUSTIN_CASE_WALLET,        889);
+        DssExecLib.sendPaymentFromSurplusBuffer(GFX_LABS_WALLET,           641);
+
+
+        //////////////////////////////////////////////////////////
+        // Repair Dai Streams
+        // https://forum.makerdao.com/t/correction-to-last-weeks-executive/13022
+
+        // MIP40c3-SP47: Core Unit Budget (SNE-001) - Phase II StarkNet Fast Withdrawal and Wormhole
+        // https://mips.makerdao.com/mips/details/MIP40c3SP47
+        uint256 _sneId = 24;
+        // Send first month payment minus accrued amount
+        uint256 snePayment = sub(42_917 * WAD, VEST.accrued(_sneId));
+        VatAbstract(MCD_VAT).suck(MCD_VOW, address(this), snePayment * RAY);
```

Comment:
I think this tripped Naz up and I just found myself verifying it. Maybe just a quick comment here that `WAD * RAY == RAD`

```suggestion
        VatAbstract(MCD_VAT).suck(MCD_VOW, address(this), snePayment * RAY);  // WAD * RAY == RAD
```